queries/markdown/highlights.scm

```diff
@@ -34,6 +34,24 @@
   (info_string (language) @conceal
   (#set! conceal "")))
 
+;; Conceal bullet points
+([(list_marker_plus) (list_marker_star)]
+  @punctuation.special
+  (#offset! @punctuation.special 0 0 0 -1)
+  (#set! conceal "•"))
+([(list_marker_plus) (list_marker_star)]
+  @punctuation.special
+  (#any-of? @punctuation.special "+" "*")
+  (#set! conceal "•"))
+((list_marker_minus)
+  @punctuation.special
+  (#offset! @punctuation.special 0 0 0 -1)
+  (#set! conceal "—"))
+((list_marker_minus)
+  @punctuation.special
+  (#eq? @punctuation.special "-")
+  (#set! conceal "—"))
```

Comment:
As there can be arbitrary amount of whitespaces after `-*+`, can something like this be better?

```suggestion
((list_marker_minus)
  @punctuation.special
  (#gsub! @punctuation.special "(.)%s*" "%1")
  (#set! conceal "—"))
```

Reply:
nvm I misunderstood what the code does